src/elements/toniq-dropdown/toniq-dropdown.element.ts

```diff
@@ -0,0 +1,166 @@
+import {assign, css, defineElementEvent, html} from 'element-vir';
+import {ChevronDown24Icon} from '../../icons';
+import {noUserSelect, toniqFontStyles} from '../../styles';
+import {applyBackgroundAndForeground, toniqColors} from '../../styles/colors';
+import {defineToniqElement} from '../define-toniq-element';
+import {ToniqIcon} from '../toniq-icon/toniq-icon.element';
+
+declare global {
+    interface Window {
+        dropdownListenerAdded: any;
+    }
+}
+window.dropdownListenerAdded = window.dropdownListenerAdded || false;
+
+export interface ToniqDropdownOption {
+    value: unknown;
+    label: string;
+}
+
+export const ToniqDropdown = defineToniqElement({
+    tagName: 'toniq-dropdown',
+    props: {
+        list: [] as Readonly<ToniqDropdownOption[]>,
+        select: undefined as undefined | Readonly<ToniqDropdownOption>,
+        dropdownOpen: false,
+    },
+    events: {
+        selectChange: defineElementEvent<ToniqDropdownOption>(),
+    },
+    styles: css`
+        :host {
+            ${toniqFontStyles.boldParagraphFont};
+        }
+
+        .dropdown {
+            width: 288px;
+            height: 48px;
+            position: relative;
+            border: 0px transparent;
+            border-radius: 8px;
+            font: inherit;
+            padding: 0;
+            margin: 0;
+            color: inherit;
+        }
+
+        toniq-icon 
+            transition: all 0.2s linear;
```

Comment:
Use `interactionDuration` from `animation.ts`:
```suggestion
            transition: ${interactionDuration} linear;
```